packages/pds/src/api/com/atproto/sync.ts

```diff
@@ -1,9 +1,9 @@
 import { Server } from '../../../lexicon'
 import { InvalidRequestError } from '@atproto/xrpc-server'
-import { def as common } from '@atproto/common'
 import { Repo } from '@atproto/repo'
 import SqlRepoStorage from '../../../sql-repo-storage'
 import AppContext from '../../../context'
+import { CID } from 'multiformats/cid'
 
 export default function (server: Server, ctx: AppContext) {
   server.com.atproto.sync.getRoot(async ({ params }) => {
```

Comment:
another bikeshed, thoughts on switching this to `getHead`? 

I've done this in the storage interfaces to avoid confusion with the similarly named `root` (which refers to the ipld node that the commit points to).

Reply:
actually i just decided to do it. lmk if you have any beef with it 🐮

Reply:
Yeah I'm cool with that